packages/hathor-rpc-handler/__tests__/rpcMethods/sendTransaction.test.ts

```diff
@@ -61,6 +61,11 @@ describe('sendTransaction', () => {
             address: 'testAddress',
             token: '00',
           }],
+          outputs: [{
+            address: 'testAddress',
+            value: BigInt(100),
```

Comment:
Why not `100n`?

Reply:
Refactored everywhere to use `n` instead of `BigInt`